addons/sale/models/sale.py

```diff
@@ -71,7 +71,7 @@ def _get_invoiced(self):
             ])
             if domain_inv:
                 refund_ids = self.env['account.invoice'].search(expression.AND([
-                    ['&', ('type', '=', 'out_refund'), ('origin', '!=', False)], 
+                    ['&', ('type', '=', 'out_refund'), ('origin', '!=', False)],
```

Comment:
Minimal diff please
Remove this change

Reply:
@moylop260

Could you review this PR, please?

Regards